README.md

```diff
@@ -1,19 +1,63 @@
 # HWSC-File-Transaction-SVC
 This is one of the microservices in the Humpback Whale Social Call project, which allows user to manage their files. 
-### Purpose
 
+### Purpose
 This provides file management service which includes the following:
-  - Downloading
-  - Uploading
-  - Deleting
+- Downloading
+- Uploading
+- Deleting
+
 ## Prerequisites
-- Python 3.7
+- Python 3.7 or greater
+- pip
+- pipenv
 - Webstorm
 
-## How to install Dependencies
- - TO-DO
+## Check your python version
+- On your terminal, type [python --version] to check the python version
+
+## How to install Python3.7 with Homebrew
+- If you don't have python version 3.7, you can install it with Homebrew
+- On your terminal, type [ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"] to install Homebrew
+- Insert Homebrew directory into your path by typing [sublime ~./profile]. sublime can be substituted with your favorite text editor
```

Comment:
This line is probably misleading. `$ sublime ~./profile` will only work if you set the keyword `sublime` to the path to where your sublime is located. I would remove this line and on line 23, just say open `~./profile` with vim or your favorite text editor etc.